parse/ConditionPythonParser.cpp

```diff
@@ -10,6 +10,50 @@
 #include "ValueRefPythonParser.h"
 
 condition_wrapper operator&(const condition_wrapper& lhs, const condition_wrapper& rhs) {
+    std::shared_ptr<Condition::ValueTest> lhs_cond = std::dynamic_pointer_cast<Condition::ValueTest>(lhs.condition);
+    std::shared_ptr<Condition::ValueTest> rhs_cond = std::dynamic_pointer_cast<Condition::ValueTest>(rhs.condition);
+
+    if (lhs_cond && rhs_cond) {
+        const auto lhs_vals = lhs_cond->ValuesDouble();
+        const auto rhs_vals = rhs_cond->ValuesDouble();
+
+        if (lhs_vals[2] == nullptr && rhs_vals[2] == nullptr && lhs_vals[1] && rhs_vals[0] && (*lhs_vals[1]) == (*rhs_vals[0])) {
```

Comment:
why sometimes `if (lhs_vals[X] == nullptr)` and sometimes just `if (lhs_vals[X])` but neither `if (!lhs_vals[X])` nor `if (!lhs_vals[X])` ?


than `if (lhs_vals

Reply:
Removed comparison in favor of boolean conversion.